src/processor/reconnection-graph.service.ts

```diff
@@ -8,13 +9,12 @@ import { Queue } from 'bullmq';
 import { SubmittableExtrinsic } from '@polkadot/api-base/types';
 import { AnyNumber, ISubmittableResult } from '@polkadot/types/types';
 import { KeyringPair } from '@polkadot/keyring/types';
-import { hexToU8a, u8aToHex } from '@polkadot/util';
+import { hexToU8a } from '@polkadot/util';
 import { EventEmitter2 } from '@nestjs/event-emitter';
 import { Option, Vec } from '@polkadot/types';
-import { Hash } from '@polkadot/types/interfaces';
-import { ITxMonitorJob } from '#app/interfaces/monitor.job.interface';
-import { MILLISECONDS_PER_SECOND } from 'time-constants';
-import { BlockchainConstants } from '#app/blockchain/blockchain-constants';
+import { ITxStatus } from '#app/interfaces/tx-status.interface';
```

Comment:
Side note: This file has `TxHashMap` and `JobTxList` which appear to be unused now.

Reply:
Removed.